cpp/benchmarks/io/orc/orc_reader_options.cpp

```diff
@@ -48,7 +53,7 @@ void BM_orc_read_varying_options(nvbench::state& state,
 {
   cudf::rmm_pool_raii rmm_pool;
 
-  auto constexpr num_chunks = 1;
+  auto constexpr num_chunks = RowSelection == row_selection::ALL ? 1 : 8;
```

Comment:
Why 8?

Reply:
No specific reason, just a value that's high enough for any overhead from chunked reading to presumably show up.